### PR TITLE
fix: Show model preview runtime errors

### PR DESCRIPTION
**Pull request by skokenes**
Previously, we were not displaying runtime errors in the modeler that may be triggered when the model gets the table preview. See https://github.com/rilldata/rill-developer/issues/1570

This PR adds a check for the runtime errors at the ModelBody level and displays the runtime error if it exists, along with any other errors.

image

### CODE REVIEW

File: web-common/src/features/models/workspace/ModelBody.svelte [hunk 1]
```diff
@@ -44,6 +45,8 @@
   export let modelName: string;
   export let focusEditorOnMount = false;
 
+  let tableQuery;
```
Comment (djbarnwal) [inline web-common/src/features/models/workspace/ModelBody.svelte:48]: This is redundant. The reactive statement `$: tableQuery = ..` will initialize the variable for us.

File: web-common/src/features/models/workspace/ModelBody.svelte [hunk 2]
```diff
@@ -53,6 +56,14 @@
   $: runtimeInstanceId = $runtime.instanceId;
   const updateModel = createRuntimeServicePutFileAndReconcile();
 
+  let limit = 150;
```
Comment (djbarnwal) [inline web-common/src/features/models/workspace/ModelBody.svelte:59]: Let's make this `const` if we are not changing this anywhere

File: web-common/src/features/models/workspace/ModelBody.svelte [hunk 3]
```diff
@@ -53,6 +56,14 @@
...
+
+  $: tableQuery = createQueryServiceTableRows(runtimeInstanceId, modelName, {
+    limit,
+  });
+
+  $: runtimeError = $tableQuery.error?.response.data;
```
Comment (djbarnwal) [inline web-common/src/features/models/workspace/ModelBody.svelte:65]: I believe the type for `response` is not present. There's already a TODO note in the file. For now we can just use `($tableQuery.error as any)?.response.data`

**Review by djbarnwal — APPROVED** [review]
Looks good, have left some nit comments. Feel free to merge after those have been resolved. Approving.

File: web-common/src/features/models/workspace/ModelBody.svelte (hunk 3 above)
Comment (skokenes) [reply web-common/src/features/models/workspace/ModelBody.svelte]: Done

File: web-common/src/features/models/workspace/ModelBody.svelte (hunk 2 above)
Comment (skokenes) [reply web-common/src/features/models/workspace/ModelBody.svelte]: Done

File: web-common/src/features/models/workspace/ModelBody.svelte (hunk 1 above)
Comment (skokenes) [reply web-common/src/features/models/workspace/ModelBody.svelte]: Done

**Pull request merged**